sidecars/clang/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM debian:10-slim
```

Comment:
Would be good to update to bustier but could be done later

Reply:
:+1: for doing it later, I don't want to introduce breakages here by accident